app/models/appointment.rb

```diff
@@ -1,5 +1,34 @@
 class Appointment < ActiveRecord::Base
 	belongs_to :doctor
 	belongs_to :patient
-	validates_presence_of :doctor
```

Comment:
btw, you should either have replied my similar comment on earlier PR or it should have been addressed in this PR.